.circleci/config.yml

```diff
@@ -356,7 +356,7 @@ jobs:
       - run:
           name: "Run golangci-lint"
           no_output_timeout: 30m
-          command: golangci-lint run --new-from-rev=dev --presets format,performance -v --timeout 10m
+          command: golangci-lint run --new-from-rev=dev --presets format,performance -v --fix --timeout 10m
```

Comment:
Is this fix flag fixes the mismatch automatically? If so, it's great!

Reply:
yep, it fix the mismatch automatically, you can check the log about it

Reply:
How is the auto-fix in the CI useful?
It fixes the code inside CI machine, but does not fix the code inside our repository.
Our code will stay unpretty and we don't even notice because the CI silently succeeds.

Reply:
@Sotatek-TinnNguyen I have checked again and it's true that the code is fixed inside CI machine only. I'll update the ci-failed file.

Reply:
Updated! `coverage` and `linters` job also tested.

Reply:
@Sotatek-TinnNguyen Could you remove --fix options?

Reply:
@yoomee1313 @aidan-kwon I have removed the fix flag in lint. If the test pass, I think it's all from my side